conftest.py

```diff
@@ -190,6 +190,46 @@ def repository(tmpdir):
         yield os.path.realpath(project_path)
 
 
+@pytest.fixture()
+def template():
+    """Yield template data."""
+    template = {
+        "url": "https://github.com/SwissDataScienceCenter/renku-project-template",
+        "id": "python-minimal",
+        "index": 1,
+        "ref": "0.1.11",
+        "metadata": {"description": "nodesc"},
+    }
+
+    yield template
+
+
+@pytest.fixture()
+def project_init(template):
+    """Yield template data."""
+    data = {
+        "test_project": "test-new-project",
+        "test_project_alt": "test-new-project-2",
+    }
+
+    commands = {
+        "init": ["init", "."],
+        "init_test": ["init", data["test_project"]],
+        "init_alt": ["init", data["test_project_alt"]],
+        "remote": ["--template-source", template["url"], "--template-ref", template["ref"]],
+        "id": ["--template-id", template["id"]],
+        "index": ["--template-index", template["index"]],
+        "force": ["--force"],
+        "list": ["--list-templates"],
+        "parameters": ["--parameter", "p1=v1", "--parameter", "p2=v2"],
+        "parameters_equal_missing": ["--parameter", "p3:v3"],
+        "parameters_equal_early": ["--parameter", "=p4v3"],
+        "confirm": len(set(template["metadata"].keys())) * "\n",
+    }
+
+    yield template, data, commands
```

Comment:
You don't need to yield `template`, you can just add it to the arguments of the tests and it will reuse the same fixture.